nsxt/segment_port_common.go

```diff
@@ -396,9 +396,17 @@ type tier1SegmentPort struct {
 	ids            *segmentPort
 }
 
-func nsxtPolicySegmentPortProfilesRead(d *schema.ResourceData, m interface{}) error {
+func nsxtPolicySegmentPortProfilesRead(d *schema.ResourceData, m interface{}, isBindingResource bool) error {
```

Comment:
Instead of using the bool parameter, could use `d.GetOk()` to retrieve `segment_path`, and if not found, try `segment_port_path`. If neither found, raise an error.